src/main/scala/com/geirsson/codegen/Codegen.scala

```diff
@@ -140,8 +140,31 @@ case class Codegen(options: CodegenOptions, namingStrategy: NamingStrategy) {
     s"""|package ${options.`package`}
         |${options.imports}
         |
+        |/**
+        | * Generated using [[https://github.com/olafurpg/scala-db-codegen scala-db-codegen]]
+        | *  - Number of tables: ${tables.size}
+        | *  - Database URL: ${options.url}
+        | *  - Database schema: ${options.schema}
+        | *  - Database user: ${options.user}
+        | */
         |//noinspection ScalaStyle
         |object Tables {
+        |
+        |  /**
+        |    * Quill used to have this trait before v1, but it's still useful to keep.
+        |    * Examples are: pattern matching on wrapped type and conversion to JSON objects.
+        |    */
+        |  trait WrappedValue[T] extends Any with WrappedType { self: AnyVal =>
```

Comment:
A separate file maybe? I think a separate project is a bit overkill for only 7 lines of code. If we generate two files (one dynamic and one static) in the same package, then the generated code is fully stand-alone valid Scala code and users won't need to manually fix imports and such. What do you think?